MigrationGuide.md

```diff
@@ -1,21 +1,30 @@
-# Breaking changes
+# Migration guide
```

Comment:
Where will be the link to Migration guide? Should it be in README.md? Will it be visible in gitbook?

Reply:
You're right. Finally I decided to move migration guide to gitbook.